packages/backend/src/database/migrations/20250122161125_add_spotlight_visibility_column_to_catalog_search.ts

```diff
@@ -0,0 +1,15 @@
+import { Knex } from 'knex';
+
+const CatalogTableName = 'catalog_search';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.schema.alterTable(CatalogTableName, (table) => {
+        table.boolean('spotlight_show').defaultTo(true).notNullable();
```

Comment:
optimization suggestion: 
if you're planning to filter by this, we should also add an index

if you're not planning to filter by this, we could make/use a jsonb column, so we don't need to add migrations for all the flags

Reply:
done here! https://github.com/lightdash/lightdash/pull/13381/commits/1dbdbde9b4e4f416cdcdaeea2726f79ae43a15b5